test/hooks.test.js

```diff
@@ -1,103 +1,120 @@
 import React, { useState, useEffect } from "react"
 import { observer, Observer, useLocalStore, useAsObservableSource } from "../src"
-import { sleepHelper } from "."
 import renderer, { act } from "react-test-renderer"
 
 test("computed properties react to props when using hooks", async () => {
-    const seen = []
+    jest.useFakeTimers()
 
-    const Child = ({ x }) => {
-        const props = useAsObservableSource({ x })
-        const store = useLocalStore(() => ({
-            get getPropX() {
-                return props.x
-            }
-        }))
+    try {
+        const seen = []
 
-        return <Observer>{() => (seen.push(store.getPropX), <div>{store.getPropX}</div>)}</Observer>
-    }
+        const Child = ({ x }) => {
+            const props = useAsObservableSource({ x })
+            const store = useLocalStore(() => ({
+                get getPropX() {
+                    return props.x
+                }
+            }))
+
+            return (
+                <Observer>
+                    {() => (seen.push(store.getPropX), <div>{store.getPropX}</div>)}
+                </Observer>
+            )
+        }
 
-    const Parent = () => {
-        const [state, setState] = useState({ x: 0 })
-        seen.push("parent")
-        useEffect(() => {
-            setTimeout(() => {
-                act(() => {
-                    setState({ x: 2 })
+        const Parent = () => {
+            const [state, setState] = useState({ x: 0 })
+            seen.push("parent")
+            useEffect(() => {
+                setTimeout(() => {
+                    act(() => {
+                        setState({ x: 2 })
+                    })
                 })
-            }, 100)
-        }, [])
-        return <Child x={state.x} />
-    }
+            }, [])
+            return <Child x={state.x} />
+        }
 
-    let wrapper
-    act(() => {
-        wrapper = renderer.create(<Parent />)
-    })
-    expect(wrapper.toJSON()).toMatchInlineSnapshot(`
+        let wrapper
+        act(() => {
+            wrapper = renderer.create(<Parent />)
```

Comment:
Why it's not using `render` from RTL here? Then it wouldn't be necessary to wrap into `act`.

Reply:
That's what I already have in mind. I would like to reconsider the using of `react-test-renderer` in the future PRs.